packages/common-all/src/utils/index.ts

```diff
@@ -52,6 +52,20 @@ import {
 } from "../types/configs/dendronConfigLegacy";
 import { DVault } from "../types/DVault";
 
+export {
```

Comment:
if we are re-exporting this from common-all to other packages, maybe we should export them as a whole to prevent name collision?

Reply:
on second thought, that might complicate things a bit. Probably fine until we actually have problems with the identifiers collide.

Reply:
Exporting them directly we communicate that they are part of a [Shared Kernel](http://ddd.fed.wiki.org/view/shared-kernel) rather something extra (like under the neverthrow namespace). But I guess we are doing that already by making it part of `common-all`. From an ergonimics POV I like not having to first import a symbol and then needing to destructure it to access the parts. All having approved the current state I will merge this now as it is.